modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/Swift5ClientCodegen.java

```diff
@@ -84,6 +85,7 @@
     protected boolean useBacktickEscapes = false;
     protected boolean generateModelAdditionalProperties = true;
     protected boolean hashableModels = true;
+    protected boolean mapFileBinaryToData = true;
```

Comment:
@aymanbagabas thanks for taking the time to finish this.
I didn't had the time to do it.
Could you please make this `false` by default to start?
Because this will break the file upload feature.
When the file upload feature works with `Data` we can change this to true.